tests/parser/SaltTableTests.cpp

```diff
@@ -53,26 +56,40 @@ BOOST_AUTO_TEST_CASE( Brine ) {
         "1 1/\n"
         "\n"
         "PVTWSALT\n"
-        " 1000/\n"
```

Comment:
When you are making changes to the testdata you could consider using the support for multiline string literals in C++:
```
std::string deckData = R"(
Her comes the 
deck;

without any \n
or " characters

much easier to work with!
)";
```

Reply:
Thanks for the tip! :)